s/cptypes-lattice.ss

```diff
@@ -285,9 +288,13 @@
     (define flvector*-mask             #b0100000000000000)
     (define box-mask                   #b1000000000000000)
 
+    ;These two are more tricky, becuase they are not a constant property.
```

Comment:
These two are trickier, because they are not constant properties.

Reply:
Updated - thanks